android-sdk/src/main/java/com/checkout/android_sdk/FramesLogger.kt

```diff
@@ -120,6 +120,15 @@ internal class FramesLogger {
         )
     }
 
+    fun sendCheckoutApiClientInitialisedEvent() {
+        internalAnalyticsEvent(
+            FramesLoggingEvent(
+                INFO,
+                FramesLoggingEventType.CHECKOUT_API_CLIENT_INITIALISED
+            )
+        )
+    }
+
```

Comment:
Can you please add unit test to check if the `sendCheckoutApiClientInitialisedEvent` is called when required.